### PR TITLE
rb532: split up DEVICE_TITLE

### PR DESCRIPTION
**Pull request by mwarning**
DEVICE_TITLE is split up into DEVICE_VENDOR, DEVICE_MODEL and DEVICE_VARIANT

### CODE REVIEW

**Comment by mwarning** [comment]
@adrianschmutzler

**Comment by adrianschmutzler** [comment]
Looks good, but I can't really approve it, someone else should have a look.

File: target/linux/rb532/image/Makefile [hunk 1]
```diff
@@ -36,9 +36,11 @@ define Build/lzma-loader-elf
 endef
 
 define Device/nand
+  DEVICE_VENDOR := MikroTik
+  DEVICE_MODEL := RouterBOARD 532
+  DEVICE_VARIANT := NAND
```
Comment (dangowrt) [inline target/linux/rb532/image/Makefile]: From what I can see on [WikiDevi](https://wikidevi.com/wiki/MikroTik_RouterBOARD_532_(RB532)) this board always got NAND flash for storing the OS. Hence I'd drop the DEVICE_VARIANT as there are no other variants.

Comment (mwarning) [reply target/linux/rb532/image/Makefile]: ack

Comment (adrianschmutzler) [reply target/linux/rb532/image/Makefile]: Fine with me, too.

**Comment by mwarning** [comment]
@adrianschmutzler since the maintainer (Roman Yeryomin <roman@advem.lv>) has no github account, should we send the patch via mailing list?

**Comment by adrianschmutzler** [comment]
Yes, why not?

**Comment by hauke** [comment]
Thank you for your patch, it was applied to master.

**Pull request closed without merging**